### ISSUE TITLE
Image not accessible

### ISSUE THREAD

**Issue by StefanJanssen95** [NuGet/Home#13481]
https://learn.microsoft.com/en-us/nuget/consume-packages/package-source-mapping#package-pattern-precedence
This image is not accessible for people who are colorblind. I would recommend using a blue color for the blocks that are currently (I'm guessing) green.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 15617693-33df-025b-26c3-2d80e4df68d3
* Version Independent ID: 15617693-33df-025b-26c3-2d80e4df68d3
* Content: [Package Source Mapping](https://learn.microsoft.com/en-us/nuget/consume-packages/package-source-mapping)
* Content Source: [docs/consume-packages/Package-Source-Mapping.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/main/docs/consume-packages/Package-Source-Mapping.md)
* Service: **nuget**
* GitHub Login: @nkolev92
* Microsoft Alias: **nikolev**

**Comment by nkolev92** [comment]
We can probably use things like checkmarks and x's as well where appropriate.

**Comment by donnie-msft** [comment]
Looks like the same as https://github.com/NuGet/docs.microsoft.com-nuget/issues/3105

**Comment by StefanJanssen95** [comment]
> We can probably use things like checkmarks and x's as well where appropriate.

I dont think that would be the best solution. in both Finland and Japan (and probably more cultures) they interpret a checkmark as "wrong".
From Wikipedia: [Check mark](https://en.wikipedia.org/wiki/Check_mark#International_differences)
> In Finnish, ✓ stands for väärin, i.e., "wrong", due to its similarity to a slanted v.[[citation needed](https://en.wikipedia.org/wiki/Wikipedia:Citation_needed)] The opposite, "correct", is marked with ⋅ / ⋅ \cdot \!/\!\cdot , a slanted vertical line emphasized with two dots[[5]](https://en.wikipedia.org/wiki/Check_mark#cite_note-5) (see also [commercial minus sign](https://en.wikipedia.org/wiki/Commercial_minus_sign)).
> 
> In Japan, the [O mark](https://en.wikipedia.org/wiki/O_mark) is used instead of the check mark, and the X or ✓ mark are commonly used for wrong.[[6]](https://en.wikipedia.org/wiki/Check_mark#cite_note-6)

Besides the cultural differences, this image has three states so just two icons would not suffice.
